setup.py

```diff
@@ -185,9 +185,10 @@ def getCanthermExtensionModules():
             if os.path.splitext(source)[1] == '.pyx':
                 ext_modules.append(module)
 
-scripts=['cantherm.py', 'rmg.py', 'scripts/diffModels.py', 'scripts/generateFluxDiagram.py',
-         'scripts/generateReactions.py', 'scripts/mergeModels.py','scripts/sensitivity.py', 'scripts/thermoEstimator.py',
-         'testing/databaseTest.py']
+scripts=['cantherm.py', 'rmg.py', 'scripts/checkModels.py', 'scripts/convertFAME.py', 'scripts/diffModels.py',
+         'scripts/generateChemkinHTML.py', 'scripts/generateFluxDiagram.py', 'scripts/generateReactions.py',
+         'scripts/machineWriteDatabase.py', 'scripts/mergeModels.py', 'scripts/simulate.py',
+         'scripts/standardizeModelSpeciesNames.py', 'scripts/thermoEstimator.py', 'testing/databaseTest.py']
```

Comment:
Could you separate these into one item per line to help make future commits cleaner?